lib/mosaic-interactions/src/main/java/org/eclipse/mosaic/interactions/environment/LidarUpdates.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright (c) 2024 Fraunhofer FOKUS and others. All rights reserved.
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contact: mosaic@fokus.fraunhofer.de
+ */
+
+package org.eclipse.mosaic.interactions.environment;
+
+import static org.apache.commons.lang3.builder.ToStringStyle.SHORT_PREFIX_STYLE;
+
+import org.eclipse.mosaic.lib.objects.vehicle.sensor.LidarData;
+import org.eclipse.mosaic.rti.api.Interaction;
+
+import org.apache.commons.lang3.builder.EqualsBuilder;
+import org.apache.commons.lang3.builder.HashCodeBuilder;
+import org.apache.commons.lang3.builder.ToStringBuilder;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class LidarUpdates extends Interaction {
+    private static final long serialVersionUID = 1L;
+
+    /**
+     * String identifying the type of this interaction.
+     */
+    public final static String TYPE_ID = createTypeIdentifier(LidarUpdates.class);
+
+    /**
+     * Time at which the next sensor update will be sent.
+     */
+    private long nextUpdate;
+
+
+    /**
+     * List of {@link LidarData} containing LiDAR data from the simulator.
+     */
+    private final List<LidarData> updated;
+
+
+
+    public LidarUpdates(long time, List<LidarData> updated) {
```

Comment:
For me it isn't clear what exactly the list of updated `LidarData` is made-up of.

* Is it lidar data for **all** ego-vehicles at `time`?
* Is it lidar data for **one** ego-vehicle at `time`?

This kind of stuff should be documented.

As far as I get, each ego-vehicle has a LidarData-Object for each lidar-collection-time, which contains the measured point clouds, right?